cmd/controller/main.go

```diff
@@ -110,44 +134,17 @@ func main() {
 	}
 
 	imageServer := imagehandler.NewImageHandler(ctrl.Log.WithName("ImageHandler"), os.Getenv("DEPLOY_ISO"), imagesPublishAddr)
-	// why use a FileServer?
-	// 1. it streams files efficiently
-	// 2. if we cache these images, then that will be an easy change.
```

Comment:
Side note: 3. for Ironic internal cache it's important to support the 'Last Modified' header.

Side note 2: maybe we need to bypass the Ironic internal cache?

Side note 3: if we deploy the new controller into the Metal3 pod, we can potentially use `file:///` images and avoid yet another HTTP transfer.